Classes/Security/Authorization/Privilege/Node/Doctrine/PropertyConditionGenerator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace PunktDe\NodeRestrictions\Security\Authorization\Privilege\Node\Doctrine;
+
+/*
+ *  (c) 2017 punkt.de GmbH - Karlsruhe, Germany - http://punkt.de
```

Comment:
Its 2018 already :) The correct header for our open source classes should be sth like: 

```
/*
 * This file is part of the PunktDe.NodeRestrictions package.
 *
 * This package is open source software. For the full copyright and license
 * information, please view the LICENSE file which was distributed with this
 * source code.
 */
```